utils/ltop.c

```diff
@@ -1559,6 +1580,35 @@ _cmp_oststat_byspc (oststat_t *o1, oststat_t *o2)
     return -1 * ret;
 }
 
+static int
+_cmp_mdtstat_byspc (mdtstat_t *m1, mdtstat_t *m2)
+{
+    double t1 = sample_val (m1->kbytes_total, sort_tnow);
+    double f1 = sample_val (m1->kbytes_free,  sort_tnow);
+    double p1 = t1 > 0 ? ((t1 - f1) / t1)*100.0 : 0;
+    double t2 = sample_val (m2->kbytes_total, sort_tnow);
+    double f2 = sample_val (m2->kbytes_free,  sort_tnow);
+    double p2 = t2 > 0 ? ((t2 - f2) / t2)*100.0 : 0;
+    int ret = p1 == p2 ? 0 : p1 < p2 ? -1 : 1; /* ascending */
```

Comment:
```
    int ret = p1 == p2 ? 0 : p1 < p2 ? -1 : 1; /* ascending */
    return (-1 * ret);
```
Can you write the above two lines in longhand, so it's easier to read:
```
int ret;
if (p1 == p2)
    ret = 0;
else if (p1 < p2)
    ret  = 1;
else
    ret = -1;
return ret;
```

Reply:
Yep